frontend/app_flowy/lib/workspace/presentation/plugins/grid/src/widgets/cell/select_option_cell/select_option_editor.dart

```diff
@@ -156,6 +156,7 @@ class _TextField extends StatelessWidget {
             selectedOptionMap: optionMap,
             distanceToText: _editorPannelWidth * 0.7,
             tagController: _tagController,
+            onClick: () => FlowyOverlay.of(context).remove((SelectOptionTypeOptionEditor).toString()),
```

Comment:
How about create a function to return the identifier then we can reuse the identifier. For example:
static String identifier() {
    return (DateCellEditor).toString();
  }

Reply:
@appflowy 
If I understood it correctly, you're asking to create a function in the _TextField class itself for returning this particular identifier (SelectOptionTypeOptionEditor) only, right? 
In that case, would the name of the function 'identifier' be appropriate, since it returns only a particular identifier?

Reply:
Create a function identifier() for class, SelectOptionTypeOptionEditor. For example.

static String identifier() {
   return (SelectOptionTypeOptionEditor).toString();
}

So, we can call SelectOptionTypeOptionEditor.identifier()

Reply:
Ah! Got it. 
Thanks!